src/osal/lv_freertos.c

```diff
@@ -336,8 +345,11 @@ static void prvCheckMutexInit(lv_mutex_t * pxMutex)
     if(pxMutex->xIsInitialized == pdFALSE) {
         /* Mutex initialization must be in a critical section to prevent two threads
          * from initializing it at the same time. */
+#if LV_USE_OS == LV_OS_FREERTOS
         taskENTER_CRITICAL();
-
+#elif LV_USE_OS == LV_OS_FREERTOS_ESP
+        taskENTER_CRITICAL(&critSectionMux);
+#endif
```

Comment:
How about define a macro above to handle differences between FREERTOS and FREERTOS_ESP?

```c
#if LV_USE_OS == LV_OS_FREERTOS
        #define _enter_critical() /*need better name*/ taskENTER_CRITICAL();
        #define _exit_critical()
#elif LV_USE_OS == LV_OS_FREERTOS_ESP
        #define _enter_critical() taskENTER_CRITICAL(&critSectionMux);
        #define _exit_critical()
#endif
```